compose.override.yml

```diff
@@ -24,14 +10,24 @@ services:
       - ./backend/migrations:/migrations
     env_file:
       - .env
-    command: ["-path", "/migrations", "-database", "postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@postgres:$POSTGRES_PORT/$POSTGRES_DB?sslmode=disable", "up"]
+    command:
+      [
+        "-path",
+        "/migrations",
+        "-database",
+        "postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@postgres:$POSTGRES_PORT/$POSTGRES_DB?sslmode=disable",
+        "up",
+      ]
   backend:
     container_name: opengpts-backend
+    pull_policy: build
     build:
       context: backend
     ports:
       - "8100:8000" # Backend is accessible on localhost:8100
     depends_on:
+      langserve:
```

Comment:
Where is the `langserve` container defined?

Reply:
its provided by the langgraph cli